From 39abdcd47a5b14e35eb02691b9b1c9219c3e2187 Mon Sep 17 00:00:00 2001
From: Andrew Wang <andrew.wang@cloudera.com>
Date: Fri, 8 Apr 2016 13:58:01 -0700
Subject: [PATCH 1409/2748] CLOUDERA-BUILD. Update
 cloudera/test-distributed.sh for new-style grind
 project config.

Change-Id: If5938404a24c414c91055314940f1d5984fbe527
---
 cloudera/test-distributed.sh |   12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/cloudera/test-distributed.sh b/cloudera/test-distributed.sh
index 2ef7660..c6c5c0b 100755
--- a/cloudera/test-distributed.sh
+++ b/cloudera/test-distributed.sh
@@ -33,15 +33,17 @@ which grind
 # Go to project root
 cd "$DIR/.."
 
-cat > .grind_deps << EOF
-{
-        "empty_dirs": ["test/data", "test-dir", "log"],
-        "file_patterns": ["*.so"]
-}
+cat > .grind_project.cfg << EOF
+[grind]
+empty_dirs = ["test/data", "test-dir", "log"]
+file_globs = []
+file_patterns = ["*.so"]
+artifact_archive_globs = [u'**/surefire-reports/TEST-*.xml']
 EOF
 
 # Invoke grind to run tests
 grind -c ${DIR}/supertest/grind.cfg config
+grind -c ${DIR}/supertest/grind.cfg pconfig
 grind -c ${DIR}/supertest/grind.cfg test --artifacts -r 3 -e TestRM -e TestWorkPreservingRMRestart -e TestRMRestart -e TestContainerAllocation -e TestMRJobClient -e TestCapacityScheduler -e TestDelegatingInputFormat -e TestMRCJCFileInputFormat -e TestJobHistoryEventHandler -e TestCombineFileInputFormat -e TestAMRMRPCResponseId -e TestSystemMetricsPublisher -e TestNodesListManager -e TestRMContainerImpl -e TestApplicationMasterLauncher -e TestRMWebApp -e TestContainerManagerSecurity -e TestResourceManager -e TestParameterParser -e TestNativeCodeLoader -e TestRMContainerAllocator -e TestMRIntermediateDataEncryption -e TestWebApp -e TestCryptoStreamsWithOpensslAesCtrCryptoCodec -e TestDNS
 # TestDNS fails only on supertest. CDH-37451
 
-- 
1.7.9.5

